apps/api/src/campaign-application/campaign-application.controller.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { Test, TestingModule } from '@nestjs/testing'
+import { CampaignApplicationController } from './campaign-application.controller'
+import { CampaignApplicationService } from './campaign-application.service'
+import { SpyOf, autoSpy } from '@podkrepi-bg/testing'
+
+describe('CampaignApplicationController', () => {
+  let controller: CampaignApplicationController
+  let service: SpyOf<CampaignApplicationService>
+
+  beforeEach(async () => {
+    service = autoSpy(CampaignApplicationService)
```

Comment:
How is autoSpy different, than `module.get<CampaignApplicationService>(CampaignApplicationService)`?
Seems redundant to me.

Docs in NestJS also shows similar way to mock the Service classes [link](https://docs.nestjs.com/fundamentals/testing#testing-utilities)